### PR TITLE
fix(kicker): run cargo check in user environment

### PR DESCRIPTION
**Pull request by zeroqn**

### CODE REVIEW

File: kicker [hunk 1]
```diff
@@ -303,15 +303,20 @@ function manual-build() {
...
-        # NOTE: manually add rustfmt, gw-types crate needs this tool.
-        erun "cd $srcdir && CARGO_HOME=$srcdir/.cargo cargo check --locked && cd -"
         erun docker run \
             --rm \
             --env CARGO_HOME=/app/.cargo \
             --volume $srcdir:/app \
             --volume $WORKSPACE/packages/.rustup:/root/.rustup \
             --workdir /app \
-            retricsu/godwoken-manual-build:ckb2021 bash -c "rustup component add rustfmt && which rustfmt && cargo build --locked --release"
+            retricsu/godwoken-manual-build:ckb2021 cargo check --locked
```
Comment (keroro520) [inline kicker:312]: Just remove `cargo check`, L304-L312.

`cargo check` is used to download dependencies from the host machine. Running it inside a Docker container defeats its purpose. Therefore, just remove it.

**Review by keroro520 — CHANGES_REQUESTED** [review]

File: kicker (hunk 1 above)
Comment (zeroqn) [reply kicker]: Ok

File: kicker [hunk 2]
```diff
@@ -303,15 +303,13 @@ function manual-build() {
...
-        # NOTE: manually add rustfmt, gw-types crate needs this tool.
-        erun "cd $srcdir && CARGO_HOME=$srcdir/.cargo cargo check --locked && cd -"
         erun docker run \
             --rm \
             --env CARGO_HOME=/app/.cargo \
             --volume $srcdir:/app \
             --volume $WORKSPACE/packages/.rustup:/root/.rustup \
             --workdir /app \
-            retricsu/godwoken-manual-build:ckb2021 bash -c "rustup component add rustfmt && which rustfmt && cargo build --locked --release"
+            retricsu/godwoken-manual-build:ckb2021 cargo build --locked --release
```
Comment (keroro520) [inline kicker:312]: Please adapt the code comments L304-305

**Review by keroro520 — APPROVED** [review]

File: kicker (hunk 2 above)
Comment (zeroqn) [reply kicker:313]: Add rustfmt.

**Review by RetricSu — APPROVED** [review]

**Pull request merged**